sqlite/xen.x86_64.cfg

```diff
@@ -0,0 +1,7 @@
+name          = "sqlite"
+vcpus         = "1"
+kernel        = "./workdir/build/sqlite_xen-x86_64"
+ramdisk       = "./initrd.cpio"
+cmdline       = "sqlite_xen-x86_64 vfs.fstab=[ \"initrd0:/:extract::ramfs=1:\" ] -- sqlite3 -interactive"
+memory        = "256"
+type 	      	= "pv"
```

Comment:
Do not use Tab.

Actually leave this as it is because we need to correct it everywhere. All other `xen.*.cfg` files have this problem.